.github/workflows/ci.yml

```diff
@@ -89,6 +89,11 @@ jobs:
         uses: actions/checkout@v4
       - name: Get library artifact
         run: ./utils/scripts/load-binary.sh ${{ matrix.library }}
+      - name: Get nginx module
```

Comment:
I feel like `./utils/scripts/load-binary.sh cpp` should load both nginx and cpp, WDYT ?